babel_config.js

```diff
@@ -23,11 +23,5 @@ export default {
         "transform-regenerator"
       ],
     }]
-  ],
-  plugins: [
-    // Shipped in Firefox 57, Chrome 63
-    "@babel/plugin-proposal-async-generator-functions",
-    // Shipped in Firefox 55, Chrome 60
-    "@babel/plugin-proposal-object-rest-spread"
```

Comment:
I'm not really sure why we included these plugins here. I introduced them in #193, but then I don't know why I didn't remove them in #225 a few weeks later. After all, it would seem that the `preset-env` config would have sufficed?